docs/git-comparison.md

```diff
@@ -17,10 +17,13 @@ various use cases.
 * **The working copy is automatically committed.** That results in a simpler and
   more consistent CLI because the working copy is now treated like any other
   commit. [Details](working-copy.md).
-* **There's no index (staging area).** That also results in a simpler
-  CLI for similar reasons. The index is very similar to an intermediate commit
-  between `HEAD` and the working copy, so workflows that depend on it can be
-  modeled using proper commits instead. [Details](#the-index).
+* **There's no index (staging area).** Because the working copy is automatically
+  committed, an index-like concept doesn't make sense. The index is very similar
+  to an intermediate commit between `HEAD` and the working copy, so workflows
+  that depend on it can be modeled using proper commits instead. Jujutsu has
+  excellent support for moving changes between commits, and does so in a more
+  generic way because the commands work on arbitrary commits.
```

Comment:
Minor/optional: I am not fond of the second half of this sentence about the "generic way". I think we could just delete it. We could also say something like ", see for example the `-i` option to `jj squash` and `jj move` commands".

Reply:
I don't mind deleting it, but why do you dislike it?

Reply:
It's pretty vague; it advertises that "jj" is better here while giving general words instead of any specific evidence. So, I'm suggesting we replace the ad with the evidence (examples) and let the reader draw conclusions.

There is also the possibility of having both the general statement and one or more examples.

Again, I don't feel strongly about this, it's OK already.

Reply:
Okay, I've removed that part. Thanks for reviewing.